flocker/control/test/test_persistence.py

```diff
@@ -249,3 +252,32 @@ def test_complex_keys(self):
                                manifestations={}, paths={},
                                devices={uuid4(): FilePath(b"/tmp")})
         self.assertEqual(node_state, wire_decode(wire_encode(node_state)))
+
+
+class ConfigurationMigrationTests(SynchronousTestCase):
+    """
+    Tests for ``_ConfigurationMigration``.
+    """
+    configurations_dir = FilePath(__file__).sibling('configurations')
+
+    def test_v0_v1_configuration(self):
+        """
+        A V0 JSON configuration blob is transformed to a V1 configuration
+        blob, with the result validating when loaded.
+        """
+        v0_json = self.configurations_dir.child(
+            'configuration_v0.json').getContent()
+        v1_json = migrate_configuration(0, 1, v0_json)
+        v1_config = wire_decode(v1_json)
+        self.assertEqual(v1_config, Deployment(nodes=frozenset()))
```

Comment:
... although "and it comes up empty" probably isn't the *most* useful assertion to make.

@exarkun can you give some examples of what this sort of test would contain?